rotkehlchen/tests/api/test_history.py

```diff
@@ -220,8 +220,8 @@ def test_query_history_remote_errors(rotkehlchen_api_server_with_exchanges):
     warnings = rotki.msg_aggregator.consume_warnings()
     assert len(warnings) == 0
     errors = rotki.msg_aggregator.consume_errors()
-    assert len(errors) == 3
-    assert 'Etherscan API request http://someurl.com returned invalid JSON response: [{' in errors[0]  # noqa: E501
+    assert len(errors) == 5
+    assert 'Etherscan API request http://someurl.com returned invalid JSON response: [{' in errors[2]  # noqa: E501
```

Comment:
Hm ... what are errors[0] and errors[1] here now? And why are you not testing for them?

Reply:
They are error related to the kraken api calls. We were missing a remote error catch and I added it with an error message